src/Lumi/Components/Modal.purs

```diff
@@ -180,7 +182,7 @@ dialog = makeStateless dialogComponent render
               , children = [ props.children ]
               }
         , internalBorders: false
-        , title: ""
+        , title: mempty
```

Comment:
Bit of a nit but I think I prefer `""` here, as it gives you more information; without checking the type of `modal` I wouldn't be able to know whether this is `String` or `JSX` or some other `Monoid` type otherwise.

Reply:
Oops, I just spotted that this is JSX now. Never mind!